tests/test_tools.py

```diff
@@ -2174,5 +2179,105 @@ def test_get_tags_invalid():
         get_tags(coll)
         assert e_info == 'ERROR: valid tags are comma or space separated strings of tag names'
 
+@pytest.mark.parametrize(
+    "repo_information, expected", [
+        ({"repos": {
+            "repo1": {
+                "params": {
+                    "namespace_id": "35",
+                    "initialize_with_readme": "false",
+                    "name": "repo name"},
+                "url": "https://gitlab.thebillingegroup.com/api/v4/projects/"}}
```

Comment:
Let's use something less specific.  We don't really want to give away this much info in the open repo. Since you are mocking on the test it can be any valid url.  Same goes below.

Reply:
Done! changed to example url